### PR TITLE
Update appimagelauncher, make use of new anda directives

### PR DESCRIPTION
**Pull request by korewaChino**
Signed-off-by: Cappy Ishihara <cappy@cappuchino.xyz>

**What software is being packaged here?**
The name of the software and link to website (if it exists) and source code (if available).

**Describe the motivation**
A clear and concise description of why you want this package. Would this provide value to other users?

**Additional context**
Add any other context about the package submission here. Link to any relavent issues.

### CODE REVIEW

File: anda.hcl [hunk 1]
```diff
@@ -1 +1,6 @@
+config {
+    strip_prefix = "anda/"
```
Comment (lleyton) [inline anda.hcl:2]: Would we have to update any of the github workflows, since the prefix has changed?

Comment (korewaChino) [reply anda.hcl:2]: No, these are all simply aliases.

**Review by lleyton — DISMISSED** [review]

**Review by lleyton — APPROVED** [review]

**Comment by nothingneko** [comment]
LGTM!!

**Review by nothingneko — APPROVED** [review]
LGTM

**Review by windowsboy111 — APPROVED** [review]

**Pull request merged**